prusti-encoder/src/encoders/spec.rs

```diff
@@ -46,6 +46,26 @@ where
     })
 }
 
+pub fn is_function_trusted(def_id: DefId) -> bool {
+    with_proc_spec(def_id, |def_spec: &ProcedureSpecification| {
+        def_spec.trusted.extract_inherit().unwrap_or_default()
+    })
+    .unwrap_or_default()
+}
+
+pub fn is_type_trusted(ty: prusti_rustc_interface::middle::ty::Ty) -> bool {
```

Comment:
```suggestion
pub fn is_type_trusted(ty: ty::Ty) -> bool {
```

And then just use an import like the one that's commented out on L3 in this file.

Reply:
done, thanks